lib/fuzion/std/fileio.fz

```diff
@@ -0,0 +1,9 @@
+public fuzion.std.fileio is
+
+    public readFile(path string, pos i32) =>
```

Comment:
please add a comment before every feature explaining its purpose, if needed, the arguments. Comments for arguments should go before the arguments, i.e., 

```
  # read a file starting at given position into an array u8.
  # 
  # NYI: currently reads a single u8 byte only. 
  #
  public readFile(
                  # the (relative or absulute) file name, using platform specific path separators
                  path string,

                  # the byte position to read from 
                  pos i32) =>
```
In general, please use ì64` (or `u64`?) for file positions and lengths, will avoid trouble in the future! I am thinking about making `i64` the default integer type anyways.